Cargo.lock

```diff
@@ -1432,7 +1432,7 @@ dependencies = [
  "rmp-serde",
  "serde",
  "serde_json",
- "serde_with 1.14.0",
+ "serde_with 3.6.0",
```

Comment:
Expected?

Reply:
That's the version used across all the other crates (except us?), so I guess it's expected yes.